tide_automated_listing.module

```diff
@@ -4,29 +4,3 @@
  * @file
  * Tide Automated Listing component.
  */
-
-use Drupal\field\Entity\FieldConfig;
-
-/**
- * Implements hook_entity_bundle_create().
- */
-function tide_automated_listing_entity_bundle_create($entity_type_id, $bundle) {
```

Comment:
@asarfara cc @anthony-malkoun without this hook the component won't be available in landing page. The component is created to support landing page, hence the module is responsible to add its component to landing page.

Reply:
This has been moved to https://github.com/dpc-sdp/tide_landing_page/pull/105